backup.py

```diff
@@ -333,6 +353,32 @@ def main(argv):
 
             vm = vm[0]
 
+            # Get the Attachments Disk
+            disk_attachments = vms_service.vm_service(vm.id).disk_attachments_service().list()
+            disksBackup = []
+            BootableOnly = config.get_bootable_only()
+            for disk_attachment in disk_attachments:
+                if BootableOnly:
+                    if disk_attachment.bootable == True:
+                        disksBackup.append(disk_attachment)
+                        break
+                elif config.get_with_disks_deactivated() or disk_attachment.active:
+                    disksBackup.append(disk_attachment)
+
+            if not BootableOnly:
+                disks_id_exclude=config.get_disks_id_exclude()
+                if disks_id_exclude:
+                    disksBackup = [disk for disk in disksBackup if disk.id not in disks_id_exclude]
+
+            for disk_attachment in disksBackup:
+                if disk_attachment.bootable == True:
+                    logger.info("Finding bootable disk: %s", disk_attachment.id)
```

Comment:
maybe it will be better to use the new python3 format like:
```suggestion
                    logger.info(f"Finding bootable disk: {disk_attachment.id}")
```
IMHO it looks better and more readable.